# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/systemd/hotfix-syscall-ia64.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-258.3/src/include/override/sys/syscall.h.vanilla	2026-02-27 13:25:38.213699630 +0100
+++ systemd-258.3/src/include/override/sys/syscall.h	2026-02-27 13:26:52.533491878 +0100
@@ -276,7 +276,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__ia64__)
-#    define systemd_NR_open_tree_attr -1
+#    define systemd_NR_open_tree_attr 467
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_open_tree_attr 467
 #  elif defined(__m68k__)
@@ -486,7 +486,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_removexattrat 466
 #  elif defined(__ia64__)
-#    define systemd_NR_removexattrat -1
+#    define systemd_NR_removexattrat 466
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_removexattrat 466
 #  elif defined(__m68k__)
@@ -556,7 +556,7 @@
 #  elif defined(__i386__)
 #    define systemd_NR_setxattrat 463
 #  elif defined(__ia64__)
-#    define systemd_NR_setxattrat -1
+#    define systemd_NR_setxattrat 463
 #  elif defined(__loongarch_lp64)
 #    define systemd_NR_setxattrat 463
 #  elif defined(__m68k__)
